<launch>
<group ns="seneka">
  <node name="VODNODE" pkg="videoOnDemand" type="videoOnDemand">
  
	<param name="framesPerVideo"        type="int"    value="200"/>
	<param name="framesPerCache"        type="int"    value="100"/>
	<param name="framesPerBinary"       type="int"    value="100"/>
	<param name="videoFrameRate"        type="int"    value="10"/>
	<param name="binaryFilePath"        type="string" value="/home/cmm-jg/Bilder/container"/>
	<param name="videoFilePath"         type="string" value="/home/cmm-jg/Bilder/videoOnDemand.avi"/>
	<param name="showFrame"				type="bool"   value="true"/>
      
  </node>
</group>
</launch>
